packages/test/service-load-test/src/utils.ts

```diff
@@ -26,12 +26,12 @@ class FileLogger implements ITelemetryBufferedLogger {
 
     public constructor(private readonly baseLogger?: ITelemetryBufferedLogger) {}
 
-    async flush(runInfo?: {testId: string,  runId?: number}): Promise<void> {
+    async flush(runInfo?: {url: string,  runId?: number}): Promise<void> {
         const baseFlushP =  this.baseLogger?.flush();
 
         if(this.error && runInfo !== undefined) {
             const logs = this.logs;
-            const path = `${__dirname}/output/${runInfo.testId}/`;
+            const path = `${__dirname}/output/${crypto.createHash("md5").update(runInfo.url).digest("hex")}/`;
```

Comment:
Nit - For usability, maybe take a substring of the hash, like 8 or 16 chars instead of the full 32?